OKR.md

```diff
@@ -0,0 +1,50 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q2
+
+> Work in Progress
+
+### The daemon is stable and so core is reliable
+
+- `PX` - OWNER - The uncaught errors and memory usage are under control and a daemon can run for a week non stop
+- `PX` - OWNER - One or more js-ipfs daemons are part of the IPFS infrastructure
+- `PX` - OWNER - js.ipfs.io exposes an IPFS Gateway provided by a js-ipfs daemon
+
+### The core can handle vast amounts of data
+
+- `PX` - OWNER - The MFS is implemented in js-ipfs
+- `PX` - OWNER - npm on IPFS is used as a test vector of js-ipfs capabilities
+- `PX` - OWNER - js-ipfs in the browser can load a dataset of at least 100Gb (create archives.js.ipfs.io)
+- `PX` - OWNER - js-ipfs in Node.js can load a dataset of at least 1TB (use npm to prove it)
```

Comment:
Last time I checked the registry the total size was 800GB with the latest version of each package. Also, not very clear what "load a dataset" means, guessing it means load an object from a dataset, not the entire dataset.

Reply:
I meant it as `ipfs add` and `ipfs get`. Load it in and make sure when it is in, that it can also be taken out.